### PR TITLE
Fix/small

### PR DESCRIPTION
**Pull request by bobetbat**
- checkout page ui
- WIN-510
- WIN-826

### CODE REVIEW

File: src/components/AccordionBox.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const CloseIcon = () => (
+  <Iconify icon="material-symbols:expand-more-rounded" width={30} height={30} />
+);
+
+export const AccordionBox: React.FC<{ title: string; children: React.ReactNode }> = ({
+  title,
+  children
+}) => {
+  const isDesktop = useResponsive('up', 'md');
```
[the remark this answers is not in the document]
Comment (bobetbat) [reply src/components/AccordionBox.tsx:14]: not really checkout but mobile. AccordionBox can be used in any part of app, and will appear as Accordion on mobile

[the remark this answers is not in the document]
Comment (silksil) [reply src/components/AccordionBox.tsx:14]: Yes i understand that, and I see that we could potentially reusse this component for mobile. Nonehteless, I don't it is very likely that we would reuse it therefore I would personally not create a reusable component.


If you still think we should create a reusable component for this, I would consider renaming it. I believe it is a bit confusion we have a `Accordion` out of MUI and a `AccordionBox` in components. Do you think the curing naming makes it descriptive when to use which one?

[the remark this answers is not in the document]
Comment (bobetbat) [reply src/components/AccordionBox.tsx:14]: I am good with this name, the box that wraps children in accordion on mobile

[the remark this answers is not in the document]
Comment (silksil) [reply src/components/AccordionBox.tsx:14]: Is it possible to include at least in the naming that is mobile only? 

Something like `AccordionBoxMobile`

**Review by silksil — APPROVED** [review]

**Pull request merged**